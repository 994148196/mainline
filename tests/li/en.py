import numpy as np 
import matplotlib.pyplot as plt 

E=[]
Err=[]
data=np.zeros(10000)
f=open('en.out','r')
isErr=0
k=0
for line in f:
	data[k]=float(line)
	k+=1

isErr=0
for i in range(len(data)):
	en=data[i]
	if(isErr==1):
		if(en in Err):
			isErr=0
		else:
			Err.append(en)
			isErr=0
	if(en<0):
		if(en in E):
			pass
		else:
			E.append(en)
			isErr=1

plt.errorbar(np.arange(len(E)),E,yerr=Err,label="New, exact")
plt.title("Li Test")
plt.ylabel("Energy, Ha")
plt.xlabel("Iteration")
plt.legend(loc=1)
plt.show()
