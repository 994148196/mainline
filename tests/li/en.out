-1.35001
0.0239196
-1.58525
0.02045
-1.90993
0.0202576
-2.28536
0.0191527
-2.65853
0.0281176
-3.04836
0.0168175
-3.08993
0.019879
-3.30378
0.0219266
-3.38568
0.0256594
-3.37687
0.0112341
-3.37454
0.00775829
